addons/evac/functions/fn_stretcher_init.sqf

```diff
@@ -0,0 +1,4 @@
+["Attachable_Stretcher", ["CONTAINER","CLOTHES"], ["Attach Stretcher"], [], "", [{true},{params ["_unit", "_container", "_item", "_slot", "_params"];_unit == vehicle _unit}], {
```

Comment:
Same as above: `fnc_stretcher_init.sqf`

Reply:
And get a comment like this going and clean up the formatting so it's a bit more consistent and then I think should be good to go.
```
/*
 * Author: mharis001
 * Checks if AED Station can be performed on the patient.
 *
 * Arguments:
 * 0: Medic (not used) <OBJECT>
 * 1: Patient <OBJECT>
 *
 * Return Value:
 * Can CPR <BOOL>
 *
 * Example:
 * [player, cursorObject] call kat_circulation_fnc_AEDStationCondition
 *
 * Public: No
 */
```